core/src/main/java/org/apache/brooklyn/enricher/stock/Enrichers.java

```diff
@@ -891,6 +892,39 @@ protected static int count(Iterable<? extends Object> vals, boolean includeNullV
         return result;
     }
     
+    @Beta
+    public static class ComputingIsQuorate<T extends Object> implements Function<Collection<T>, T> {
```

Comment:
Had to compare this to existing enrichers to clock the logic. Why `T extends Object` and not just `Boolean`? You've got `(T) Boolean.valueOf()` in `apply` but the Boolean class is `final` so `T` can't be anything but a Boolean.